frontend/src/global_styles/content/work_packages/tabs/_files.sass

```diff
@@ -92,7 +95,7 @@
       grid-area: icon
       width: 3.375rem
       height: 3.375rem
-      font-size: 1.25rem
+      font-size: 1.75rem
```

Comment:
Although I know that is has been like this before, please do me the favour and change the classes used for this overwrite. Currently, there it is `.spot-icon.big` which I have two issues with:
1. We don't want to overwrite the spot classes in any case. It can later have unwanted side effects for other icons used in this context. Instead, please introduce a new class.
2. `big` is a modifier and should be written as those, according to BEM (so something like `new-icon-class_big`). Since you have a specific class anyway, you could also consider removing the modifier completely.